include/sparrow/layout/array_bitmap_base.hpp

```diff
@@ -19,9 +19,14 @@
 
 namespace sparrow
 {
-    /*
+    /**
      * Base class for arrays using a validity buffer for
      * defining their bitmap.
+     *
+     * @tparam D The derived type, i.e. the inheriting class for which
```

Comment:
Because at the point where we would put the reqiurement on D, the base has not be defined yet. Or I don't see which kind of requirement you're talking about.

Reply:
Ah ok `array_bitmap_base_impl` is defined below. It could be declared first and then required here. the requirement is checked at instantiation so that could work.